multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from typing import Optional
+
+import torch
+from torch import nn
+from mmdet.models import build_detector
+import mmcv
+from mmcv.runner import load_checkpoint
+import warnings
+from mmdet.core import get_classes
+from mim.commands.download import download
+
+from ..constants import AUTOMM, COLUMN, COLUMN_FEATURES, FEATURES, IMAGE, IMAGE_VALID_NUM, LABEL, LOGITS, MASKS
+from .utils import assign_layer_ids, get_column_features, get_model_head
+
+logger = logging.getLogger(AUTOMM)
+
+
+class MMdetAutoModelForObjectDetection(nn.Module):
```

Comment:
`MMdetAutoModelForObjectDetection` or `MMDetAutoModelForObjectDetection`?

Reply:
I can switch to `MMDetAutoModelForObjectDetection`